blockchain/state_transition.go

```diff
@@ -170,21 +170,30 @@ func (st *StateTransition) buyGas() error {
 	validatedSender := st.msg.ValidatedSender()
 	feeRatio, isRatioTx := st.msg.FeeRatio()
 	if isRatioTx {
-		feePayer, feeSender := types.CalcFeeWithRatio(feeRatio, mgval)
+		feePayerFee, senderFee := types.CalcFeeWithRatio(feeRatio, mgval)
 
-		if st.state.GetBalance(validatedFeePayer).Cmp(feePayer) < 0 {
+		if st.state.GetBalance(validatedFeePayer).Cmp(feePayerFee) < 0 {
+			logger.Error("%w - feePayer: %v, feePayerBalance: %v, feePayerFee: %v, txHash: %v",
```

Comment:
How about Debug log level? I think this is not an error case of Klaytn node.

And how about logging like this?

```suggestion
			logger.Debug("StateTransition buyGas failed", "err"r, errInsufficientBalanceForGasFeePayer, "feePayer",  validatedFeePayer.String() ,"feePayerBalance", st.state.GetBalance(validatedFeePayer).Uint64()...
```